iOS/ExposureNotification/source/ExposureNotification.csproj

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <PropertyGroup>
+    <TargetFrameworks>Xamarin.iOS10</TargetFrameworks>
+    <IsBindingProject>true</IsBindingProject>
+    <RootNamespace>ExposureNotification</RootNamespace>
+    <AssemblyName>Xamarin.iOS.ExposureNotification</AssemblyName>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <FileVersion>1.0.0.0</FileVersion>
+    <IPhoneResourcePrefix>Resources</IPhoneResourcePrefix>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ProcessEnums>true</ProcessEnums>
```

Comment:
What does this do? Looks new to me...

Reply:
The ApiDefinition.cs contains the enums so this allows them to be processed by the generator. This is specially useful for [smart enums](https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/binding-types-reference#enum-attributes)